### ISSUE TITLE
Major improvements

### ISSUE THREAD

**Issue by LunatiqueCoder** [nuclearpasta/react-native-drax#165]
## 🚧 WIP - This issue might change

👋 Hello @lafiosca. Thank you for all the work on this library!

Here's a list of the changes I've been working on. After we test them in production, we are willing to open pull requests and contribute to the main project.

---
### Reanimated v3
  - [ ] Replace `Animated` with `Reanimated`
  - [ ] `animateSnapForward` prop on a successful drop in a receiver view
  - [ ] Animate based on scroll position as well (for receiver and dragged views and for `animateSnapForward` and `animateSnapBack`)
  - [ ] Investigate `measure()` on the UI thread: https://docs.swmansion.com/react-native-reanimated/docs/advanced/measure
  - [ ] Fix #161 
  
 #### ▶️ Video of `Reanimated` proof of concept: https://x.com/LunatiqueCoder/status/1721982083902394694?s=20
--- 
### RN Gesture Handler v2
  - [ ] #140 
  - [ ] ☝️ While this was an easy one, we're blocked by https://github.com/software-mansion/react-native-gesture-handler/issues/2343. This issue happens when there are multiple `GestureDetectors` from the new `gesture-handler` API. Reducing the number of `GestureDetectors` will make the issue less annoying. Currently cleaning up a reproduction repo: [LunatiqueCoder/expo-drax-example](https://github.com/LunatiqueCoder/expo-drax-example) which is forked from: [nuclearpasta/react-native-drax-example](https://github.com/nuclearpasta/react-native-drax-example)

**Comment by lafiosca** [comment]
@LunatiqueCoder Thank you for your thoughtful contributions here and in other issues. As you've likely seen, I do not have the available time and energy to do a good job of maintaining this library these days. If you might be interested to become a maintainer and put your changes into a new major version release, let's have a conversation to discuss that.

**Comment by LunatiqueCoder** [comment]
@lafiosca Sure. Where would you like to continue the conversation?

**Comment by lafiosca** [comment]
@LunatiqueCoder Please send me an email about it at joe@lafiosca.com and we'll figure something out from there :)

**Comment by nhuesmann** [comment]
Has there been any progress on this? Would love to see this library updated, seems like it'd be perfect for my use case if it had current deps, but I def don't have the bandwidth to dive in.

**Comment by lafiosca** [comment]
@nhuesmann I had a great discussion with @LunatiqueCoder in December, and at that time he was hoping to get something released around the end of year, but I think that some higher priority issues came up for his team. I do not want to speak for him though, so I will let him comment here if he wants!

**Comment by LunatiqueCoder** [comment]
@nhuesmann @lafiosca Indeed, not only we had some higher priority tasks for our team, but also I have some small health issues and I'm preparing for a thoracic surgery sometime next week, so we we're drastically being delayed with our initial estimation since I am the lead developer on this. :( 

It's still on our list, but it's very hard to estimate right now... We'll keep you posted though.

**Comment by lafiosca** [comment]
Oh gosh, get well soon @LunatiqueCoder  !!!

**Comment by RobSchilderr** [comment]
> thoracic surgery

Hey @LunatiqueCoder how are things going with your health?

**Comment by LunatiqueCoder** [comment]
Hey guys. Thank you for your concerns. I'm doing just great now.

We'll **slowly** continue working on the update, but there are still no estimations or deadlines. 

- You can see the proress here: #177
